src/components/main/SignUp.tsx

```diff
@@ -37,64 +37,86 @@ export default function SignUp() {
   }
 
   const onFinish = async ({ email, password, name }: UserData) => {
-    await registerWithEmailAndPassword({ email, password, name, api });
+    await registerWithEmailAndPassword({ email, password, name });
   };
 
   return (
-    <Flex justify="center" align="center" vertical style={{ height: '100vh' }}>
-      {contextHolder}
-      <Title>Welcome!</Title>
-      <Form
-        name="signUp"
-        labelCol={{ span: 8 }}
-        wrapperCol={{ span: 16 }}
-        style={{ maxWidth: 600 }}
-        initialValues={{ remember: true }}
-        onFinish={onFinish}
-      >
-        <Form.Item<FieldType>
-          label="Email"
-          name="email"
-          rules={[
-            { required: true, message: 'Please input your email!' },
-            { type: 'email', message: 'Please enter a valid email!' },
-          ]}
+    <Content>
+      <Flex justify="center" align="center" vertical style={{ height: '75vh' }}>
+        <Title>Welcome!</Title>
```

Comment:
например

```suggestion
<Content>
      <Flex justify="center" align="center" vertical style={{ height: '75vh' }}>
        <Card
          style={{
            width: '50%',
            minWidth: '255px',
            textAlign: 'center',
            backgroundColor: 'transparent',
            border: 'solid var(--color-additional-light)',
            boxShadow: '0 4px 10px var(--color-additional)',
            padding: '50px 150px',
          }}
        >
          <Title>Welcome!</Title>
```


<img width="924" height="850" alt="image" src="https://github.com/user-attachments/assets/7f069855-845e-4f7d-bc7c-3cc2469ef33d" />

Reply:
добавил)